profiles/base.yaml

```diff
@@ -382,6 +382,7 @@ storage_containers:
 gear:
 gear_sets:
 empath_healing:
+saferoom_health_threshold: 25
```

Comment:
The method to get 25 wasn't very scientific.  I looked at myself and said 'This is about where I would go to get healing' and checked my score as 26, but you're right in that the way the scoring works it doesn't weigh bad injuries very well.  What do you think about making the scoring method use an exponential rather than a linear scale to give more weight to the worse injuries?  Something like

score = health_hash.map { |k, v| (k ** 2) * v.count }.reduce(:+)